apigee_edge.module

```diff
@@ -718,6 +718,8 @@ function apigee_edge_form_user_form_alter(&$form, FormStateInterface $form_state
   // connection validation apigee_edge_module_implements_alter() ensures that
   // form_alter hook is called in the last time.
   $validation_functions[] = 'apigee_edge_form_user_form_developer_email_validate';
+  // Validates whether the provided email address is in lowercase.
+  $validation_functions[] = 'apigee_edge_email_field_validate';
```

Comment:
For validation can be bypassed when a user is created via JSONAPI. Entity or field level constraints are the only solutions that all components should obey.

Reply:
Hi @mxr576 Sure, I will add field constraints for email field.